python/mlcroissant/mlcroissant/_src/operation_graph/operations/field.py

```diff
@@ -226,9 +229,13 @@ def _get_result(row):
                 if _is_na(value):
                     value = None
                 elif is_repeated:
-                    value = [
-                        _cast_value(self.node.ctx, v, field.data_type) for v in value
-                    ]
+                    try:
+                        value = [
+                            _cast_value(self.node.ctx, v, field.data_type)
+                            for v in value
+                        ]
+                    except TypeError:
+                        value = value
```

Comment:
What do you think of dropping the _cast_value?

Reply:
Are you sure we want to do that for all cases?